src/Emitter.ts

```diff
@@ -100,22 +113,32 @@ export class Emitter
 
     private _createTreeNodes(docs: TAnyDoclet[])
     {
+        debug(`Emitter._createTreeNodes()`);
```

Comment:
Love the debug line, great idea.

Reply:
With the docletDebugInfo() function, makes it really easier to understand the way it works :-)

Reply:
FYI, in the version of the code you read, the `debug()` function is activated through the `verbose` option (which activates the `warn()` function).
In the next commits, it will be conditionned by a specific `debug` option.